frontend/src/components/common/Filter/descendant_mappings/tissue_descendants.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{}
```

Comment:
If these are empty, can they be removed

Reply:
These are populated dynamically at build-time; if we reference them without the file existing, TS throws lint errors. We could suppress these errors, but I think it might be clearer to add a README in the directory explaining when they get populated and leaving them as-is, so future devs know this directory/files are meant to exist. I added a README, let me know what you think